src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -1498,7 +1504,10 @@ private void login(String primary,
         // for all other cases, including multiSubnetFailover
         final boolean isDBMirroring = (null == mirror && null == foActual) ? false : true;
         int sleepInterval = 100;  // milliseconds to sleep (back off) between attempts.
-        long timeoutUnitInterval;
+        long timeoutUnitInterval = 0;
```

Comment:
I use those additional variables because I couldn't find a way to decide whether TIMEOUTSTEP or TIMEOUTSTEP_TNIR needs to be used, since I only know whether TNIR is on or off after [DNS resolution](https://github.com/Microsoft/mssql-jdbc/pull/209/files#diff-0a4455f5ffd8e31e9e1badaa7822375eR1657)